### ISSUE TITLE
RSS feed for newsletters/posts

### ISSUE THREAD

**Issue by ptgolden** [OBOFoundry/OBOFoundry.github.io#2645]
It would be nice to have an RSS feed for the content in `_posts`. The "blessed" method for jekyll seems to be <https://github.com/jekyll/jekyll-feed>. I am happy to open a PR if that is a welcome addition.

**Comment by matentzn** [comment]
We would certainly be happy with a PR! Thanks @ptgolden - I did not know something like this was at all possible!

**Comment by ptgolden** [comment]
Added in #2646

**Comment by ptgolden** [comment]
The feed will show up at <https://obofoundry.org/feed.xml>, and be linked in the `<head>` after the stylesheets.

**Comment by nlharris** [comment]
Cool, thanks, @ptgolden! Once this is merged, I will test it out in feedly.

**Comment by nlharris** [comment]
Waiting for #2646 to be merged...

**Comment by ptgolden** [comment]
Awesome! I put it in my feed reader :-)
